sql/item_mods.sql

```diff
@@ -33630,14 +33652,78 @@ INSERT INTO `item_mods` VALUES (17295,26,10); -- RACC: 10
 -- Platinum Bullet
 INSERT INTO `item_mods` VALUES (17300,26,10); -- RACC: 10
 
+-- Cluster Arm
+INSERT INTO `item_mods` VALUES (17305,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17305,499,1);  -- ITEM_SUBEFFECT: SUBEFFECT_FIRE_DAMAGE
+INSERT INTO `item_mods` VALUES (17305,500,10); -- ITEM_ADDEFFECT_DMG: 10
+INSERT INTO `item_mods` VALUES (17305,501,95); -- ITEM_ADDEFFECT_CHANCE: 95
+INSERT INTO `item_mods` VALUES (17305,950,1);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_FIRE
+
+-- Snoll Arm
+INSERT INTO `item_mods` VALUES (17306,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17306,499,2);  -- ITEM_SUBEFFECT: SUBEFFECT_ICE_DAMAGE
+INSERT INTO `item_mods` VALUES (17306,500,10); -- ITEM_ADDEFFECT_DMG: 10
+INSERT INTO `item_mods` VALUES (17306,501,95); -- ITEM_ADDEFFECT_CHANCE: 95
+INSERT INTO `item_mods` VALUES (17306,950,2);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_ICE
+
+-- Grenade
+INSERT INTO `item_mods` VALUES (17313,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17313,499,1);  -- ITEM_SUBEFFECT: SUBEFFECT_FIRE_DAMAGE
+INSERT INTO `item_mods` VALUES (17313,500,10); -- ITEM_ADDEFFECT_DMG: 15
+INSERT INTO `item_mods` VALUES (17313,501,95); -- ITEM_ADDEFFECT_CHANCE: 95
```

Comment:
where is this high proc chance sourced from for these? Having thrown a lot of these\*, I seriously doubt its that high.


###### \* _I always thought it was hilarious to steal them then immediately throw it at the mob I stole it from. Same reason I sometimes eat a lizard egg while fighting a lizard. I'm just that kind of monster, beating bombs with their own detached limbs and baby killing._

Reply:
With the arrows I believe it, I'm just questioning the thrown items

Reply:
Decreased the splody thing proc rates until capture discord can confirm. I guess better lower than higher, so it doesn't become unbalanced.